modules/lang-painless/src/main/java/org/elasticsearch/painless/Locals.java

```diff
@@ -37,36 +39,14 @@
  */
 public final class Locals {
 
-    /** Reserved word: params map parameter */
-    public static final String PARAMS = "params";
-    /** Reserved word: Lucene scorer parameter */
-    public static final String SCORER = "#scorer";
-    /** Reserved word: _value variable for aggregations */
-    public static final String VALUE  = "_value";
-    /** Reserved word: _score variable for search scripts */
-    public static final String SCORE  = "_score";
-    /** Reserved word: ctx map for executable scripts */
-    public static final String CTX    = "ctx";
     /** Reserved word: loop counter */
     public static final String LOOP   = "#loop";
     /** Reserved word: unused */
     public static final String THIS   = "#this";
-    /** Reserved word: unused */
-    public static final String DOC    = "doc";
-
-    /** Map of always reserved keywords for the main scope */
-    public static final Set<String> MAIN_KEYWORDS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
-        THIS,PARAMS,SCORER,DOC,VALUE,SCORE,CTX,LOOP
-    )));
 
     /** Map of always reserved keywords for a function scope */
-    public static final Set<String> FUNCTION_KEYWORDS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
-        THIS,LOOP
-    )));
-
-    /** Map of always reserved keywords for a lambda scope */
-    public static final Set<String> LAMBDA_KEYWORDS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
-        THIS,LOOP
+    public static final Set<String> KEYWORDS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
```

Comment:
We no longer need three different types of keywords because those are now just normal variables. Well, they were normal variables before this PR as well. I think these were a leftover.